tests/utils.py

```diff
@@ -261,7 +261,8 @@ def create_object(stream):
 
         elif stream == 'invoice_line_items':
             # An invoice_line_item is implicity generated by the creation of an invoice
-           return create_object('invoices')
+            invoice = create_object('invoices')
```

Comment:
This invoice variable is not used. Why don't you use it in the subsequent line? If you don't use it intentionally, why create the variable and not just call create_object?

Reply:
Oops, good catch. Change made.